File: datastructures/java/labs/lab2_BubbleSort/ArraytoInt.java
import java.util.*;

public class ArraytoInt {
    public static void main(String[] args) {
        String year = "";
        int[] arr = { 1, 9, 8, 9 };
        // add each number in arr one by one as a string to the empty string year
        for (int i : arr) {
            year += i;
        }
        // convert the string year to an integer x
        int x = Integer.valueOf(year);
        // add 1 to x and print it to console
        System.out.println(x + 1);
    }
}

File: datastructures/python/labs/lab2_BubbleSort/ArraytoInt.py
year = ""
arr = [1, 9, 8, 9]

# add each number to an empty string
for i in arr:
    year += str(i)

# convert the string back to an integer
year = int(year)

# print the integer year and year + 1
print(year, year + 1)
